namesilo.php

```diff
@@ -2891,7 +2897,17 @@ private function formatPhone($number, $country)
             Loader::loadModels($this, ['Contacts']);
         }
 
-        return $this->Contacts->intlNumber($number, $country, '.');
+        $formatted_phone = preg_replace('/[^0-9]+/', '', $number ?? '');
```

Comment:
hmm this doesn't do anything to remove country code for places other than CA and the US.  I think one slightly more robust option would be to use intlNumber() and then exploding on the . separator and grabbing the last peice. You could still limit US and CA numbers to 10 digits

Reply:
Updated